examples/Catch/CMakeLists.txt

```diff
@@ -11,12 +11,12 @@ project(download-Catch)
 
 hunter_add_package(Catch)
 
-find_package(Catch CONFIG REQUIRED)
+find_package(Catch2 CONFIG REQUIRED)

```

Comment:
Need to update documentation about this change.

You can use this trick to avoid duplication:
* https://github.com/ruslo/hunter/blob/4ed7f821130ec2b860442a566ef2b901477fd80a/examples/foo/CMakeLists.txt#L12-L18
* https://raw.githubusercontent.com/ruslo/hunter/4ed7f821130ec2b860442a566ef2b901477fd80a/docs/packages/pkg/foo.rst